src/plugins/data_source_management/public/components/edit_data_source/components/edit_form/edit_data_source_form.tsx

```diff
@@ -930,17 +962,21 @@ export class EditDataSourceForm extends React.Component<EditDataSourceProps, Edi
       auth.type === formValues.auth.type &&
       auth.type === AuthType.UsernamePasswordType &&
       formValues.auth.credentials?.username !== auth.credentials?.username;
+    const isAuthTypeSigV4Unchanged =
+      auth.type === formValues.auth.type && auth.type === AuthType.SigV4;
     const isRegionChanged =
-      auth.type === formValues.auth.type &&
-      auth.type === AuthType.SigV4 &&
-      formValues.auth.credentials?.region !== auth.credentials?.region;
+      isAuthTypeSigV4Unchanged && formValues.auth.credentials?.region !== auth.credentials?.region;
+    const isServiceNameChanged =
```

Comment:
i needed to change the service name once to be able to save.

Reply:
No you don't. This is for the edit flow, if anything changes(title, auth type, description, etc), it will trigger the save button to render. Service name change could be one trigger, but not the mandatory

Reply:
I think I know what you mean. I'll update the create flow, to have default "service name" value set to the state when "authType" changes to SigV4.

Reply:
update with new commit